src/components/Match/matchColumns.jsx

```diff
@@ -37,7 +37,7 @@ export const heroTdColumn = {
   displayName: 'Player',
   field: 'hero_id',
   displayFn: heroTd,
-  sortFn: row => (row.level),
+  sortFn: row => row.solo_competitive_rank,
```

Comment:
Probably we should add the button that can restore the default order for table. If user used sorting, show button on right side of the table heading.

![image](https://cloud.githubusercontent.com/assets/20866892/19345022/59a9f840-9145-11e6-9a54-c171d0dbb347.png)